src/ui/purchases-ui.svelte

```diff
@@ -235,21 +234,24 @@
           );
       }
 
+      priceBreakdown.taxCollectionEnabled =
+        priceBreakdown.taxCalculationStatus !== null;
+
       return;
     }
 
+    const { data } = taxCalculation;
     priceBreakdown.taxCalculationStatus = "calculated";
-    priceBreakdown.totalAmountInMicros =
-      taxCalculation.data.total_amount_in_micros;
-    priceBreakdown.taxAmountInMicros = taxCalculation.data.tax_amount_in_micros;
+    priceBreakdown.totalAmountInMicros = data.total_amount_in_micros;
+    priceBreakdown.taxAmountInMicros = data.tax_amount_in_micros;
     priceBreakdown.totalExcludingTaxInMicros =
-      taxCalculation.data.total_excluding_tax_in_micros;
-    priceBreakdown.taxBreakdown =
-      taxCalculation.data.pricing_phases.base.tax_breakdown;
+      data.total_excluding_tax_in_micros;
+    priceBreakdown.taxBreakdown = data.pricing_phases.base.tax_breakdown;
     priceBreakdown.pendingReason = null;
+    priceBreakdown.taxCollectionEnabled = true;
```

Comment:
Don't we risk to override the setting coming from the server?
If the priceBreakdown.taxCollectionEnabled is false due to the Feature flag we might overwrite that here and also above.

Reply:
If it doesn't maybe we can add a test for that.

Reply:
It doesn't, since we only trigger a calculation when the FF and the setting are enabled. We already have a test for that